### DISCUSSION TITLE
Xournal does not use tablet pressure

### DISCUSSION THREAD

**Discussion by Squ3let0n** [xournalpp/xournalpp#3092] [Q&A]
Before it was working, but it stopped suddenly, the table is working normally in any other application and the pressure is active in xournal, I have already installed other versions of it and nada
![image](https://user-images.githubusercontent.com/83617533/116942174-7d504480-ac47-11eb-9c24-fe9a6900372f.png)

**Comment by Squ3let0n** [comment]
The application is using it as if it were a mouse
![image](https://user-images.githubusercontent.com/83617533/116942280-affa3d00-ac47-11eb-9ec4-552ff108ab02.png)
![image](https://user-images.githubusercontent.com/83617533/116942324-c2747680-ac47-11eb-972b-57afc7c320fb.png)
And it still hangs when I touch the pen on the tablet

**Reply by personalizedrefrigerator** [reply]
Please try again with [version 1.1.0+dev](https://github.com/xournalpp/xournalpp/releases/tag/nightly). This might be something that has been fixed!

> I have already installed other versions of it and nada
![](https://user-images.githubusercontent.com/46334387/116960610-c7402700-ac55-11eb-92c4-ec99fe2b3cf8.png)

What does `Settings > Input Devices` show for you (in 1.1.0+dev)? It should look something like the above.

**Comment by Technius** [comment]
On Windows, only WinTab drivers are supported at the moment (see #1346). If you're using a Windows Ink-based driver, you'll have to wait until Windows Ink support is added in GTK. In the meantime, you can use the `Enable Pressure Inference` option in version 1.1.0+dev as a workaround.
